test/Device.test.ts

```diff
@@ -33,8 +28,8 @@ suite('IoT Device Workbench: Device', () => {
       const contextMock = new TestExtensionContext();
 
       const telemetryContext: TelemetryContext = {
-        properties: {result: 'Succeeded', error: '', errorMessage: ''},
-        measurements: {duration: 0}
+        properties: {result: TelemetryResult.Succeeded, error: '', errorMessage:

```

Comment:
why not use telemetryWorker.createContext() here instead of manually construct

Reply:
This part of code is commented out. Can we just delete it?